projects/aca-content/src/lib/dialogs/node-template/create-from-template.dialog.ts

```diff
@@ -40,6 +40,7 @@ import { MatButtonModule } from '@angular/material/button';
   imports: [CommonModule, TranslateModule, MatDialogModule, ReactiveFormsModule, MatFormFieldModule, MatInputModule, MatButtonModule],
   templateUrl: './create-from-template.dialog.html',
   styleUrls: ['./create-from-template.dialog.scss'],
+  selector: 'app-create-from-template-dialog',
```

Comment:
Is this selector being used/going to be used somewhere? Also, should we change the selector being used in the [CSS](https://github.com/Alfresco/alfresco-content-app/pull/3467/files#diff-832a272b8c99e0ad8ead7c2fb7b42fbe4e37f0ef1601e47e9e5da23f2fef6ee2L1) for this component? It refers to aca-create-from-template-dialog, rather than app-create-from-template-dialog

Reply:
This component is displayed by service as it is dialog. So there is no usage of that selector in component's template directly. But we decided to add this selector because if we skip that then in HTML in browser there is 'ng-component' instead of real selector for that component (it is mentioned in jira). I discussed that with @MichalKinas and we decided that we can add selector as it will make easier finding this component in browser in html when you have real selector instead of just general ng-component. 

I used this selector in css now.